hack/run_in_cloud.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+# Note: Tested only with buildah / podman
+
+set -eux
+
+DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+if [ -z "${CNCC_REPOSITORY}" ]; then
+	echo "Please set environment variable CNCC_REPOSITORY and point it to your container repository"
+	exit 1
+fi
+
+CONTAINER_ENGINE="${CONTAINER_ENGINE:-podman}"
+BUILDAH_BIN="buildah"
+BUILDAH_BUILD_CMD="buildah bud"
```

Comment:
you can just use `podman build`, which doesn't require `buildah` to be installed